apps/speed.c

```diff
@@ -1615,6 +1622,10 @@ int speed_main(int argc, char **argv)
 
 #ifndef OPENSSL_NO_RSA
     for (i = 0; i < loopargs_len; i++) {
+        if (primes > 2) {
+            /* for multi-prime RSA, skip this */
+            break;
+        }
```

Comment:
I'm not at all sure why we should implement things this way specifically with multi-prime RSAs...  Why not add a few more test cases to `apps/testrsa.h` and add pointers to them in `rsa_data` and be done with it?

Reply:
Note that I'm just commenting here, I'm not requesting that you actually change this, I just want to understand the choices made here, as they divert quite strongly from how this program works otherwise

Reply:
Yeahok, I hear you.  I'm not certain I like it, but still, your argument makes sense, so consider the concern dropped.